web-components/src/components/form/BridgeForm.tsx

```diff
@@ -41,10 +42,13 @@ const BridgeForm = ({
     recipient: '',
     agreeErpa: false,
     batchDenom,
+    target: 'polygon', //TODO: temp?
```

Comment:
see below

Reply:
A clarification. I have kept the BRIDGE_ALLOWED_CHAINS constant on the form itself temporarily while waiting to move the forms to web-registry. In case the forms are not moved from web-components, ideally the allowed chains information should reside in the global web-registry and be passed to the (imported) form.